(docs): add low-volume 96-channel pipette specs to manual

jwwojak opened the pull request:
# Overview

This PR adds low-volume, 96-channel precision and accuracy data to the pipettes section of the manual (system description > pipettes).

RTC-628

## Test Plan and Hands on Testing

For testing and evaluation, just looking at the markdown may not convey enough of what these revisions look like. Here are some screenshots.

**Section intro**
<img width="969" height="553" alt="Screenshot 2025-09-17 at 11 27 45 AM" src="https://github.com/user-attachments/assets/a6ebe8e1-28f8-4a93-8ad7-cca95cbb50fa" />

**Sample revised table**
<img width="969" height="566" alt="Screenshot 2025-09-17 at 11 27 30 AM" src="https://github.com/user-attachments/assets/f1ff9a22-0c87-4ef9-a86a-93853075618d" />

You'll note the first column of the table. The section title and intro fit cleanly on the screen in a laptop, which is why I omitted the name "1-channel pipette" from each row.

The use of ± also matches the website.

## Changelog

**Original:** One very long table that listed specs for all the pipettes.

**Change:** 
- Revise the H2 intro, move tip and note callouts into the intro section.
- Add low vol. pipette data to appropriate section.
- Give each pipette an H3 section with a table that shows data specs. This is nice 'cause it shows up on the page-level TOC.
- Make sure the specs tables match corresponding tables on the website. What we put on the Opentrons site should be similar or match the manual. Differences in quantitative data like these specs creates uncertainty.

ps: there are typos on the website.

## Review requests

Does the new layout work. I think it is an improvement if we move specs for pipette types into separate tables instead of one big table.

## Risk assessment

Low, docs only. Let's give it a go.

jwwojak committed:
Revised intro and table pattern

jwwojak committed:
Mid-day save

- header title revisions
- intro text changes
- callout box changes
- table header changes
- add rowspan for new 96 ch data

jwwojak committed:
Make data match website tables.

- changes to H3 headers
- add intro sentence to each, repeat/reinforce pipette name.
- revise table headers
- make sure table data matches website data, esp to decimal precision
- fix spacing issues.

jwwojak committed:
Movin' things around

- H3 changes means moving callouts into the H2 intro section.
- Additional checks to make sure table data matches website.

jwwojak committed:
Minor edit

jwwojak committed:
remove line break

jwwojak committed:
Fix some tags

jwwojak committed:
Update capacity ranges to match website

- should be 5-1000, not 1-1000

ecormany commented on docs/flex/docs/system-description/pipettes.md:
```diff
@@ -40,130 +53,181 @@ Because of their wide overall range, they can use multiple sizes of tips, which
-      <td>50 µL</td>
-      <td>10 µL</td>
-      <td>1.50%</td>
+      <td>50 µL</td>
```
Oh, _weird_. I don't know how those nonstandard spaces got in this table in the first place. A quick search shows there are 15 of them remaining, even after these changes. Let's zap them and turn them into regular spaces.

jwwojak replied:
So strange. Will get rid of these.

ecormany commented on docs/flex/docs/system-description/pipettes.md:
```diff
@@ -40,130 +53,181 @@ Because of their wide overall range, they can use multiple sizes of tips, which
-      <td>1.50%</td>
+      <td>50 µL</td>
+      <td>10 µL</td>
+      <td>± 1.50%</td>
```
Dug up an old thread about this, and it seems that ± for %D is correct. Since it's a plus/minus sign, not an operator, there should be no space between it and the numeral, e.g., `±1.50%`

jwwojak replied:
Fixing those now.

ecormany commented on docs/flex/docs/system-description/pipettes.md:
```diff
@@ -40,130 +53,181 @@ Because of their wide overall range, they can use multiple sizes of tips, which
   <tbody>
     <tr>
-      <td rowspan="3"><b>Flex 1-Channel 50 µL</b></td>
+      <td rowspan="3"><b>1-50 µL</b></td>
```
Would love en dashes for these numerical ranges 😇

jwwojak replied:
Changing to en dashes –

ecormany reviewed (commented):
Couple formatting changes but the new structure and content looks good.

jwwojak reviewed (commented):
Next commit will fix reviewer-noted issues:

- Separate ranges with an en dash
- Remove the strange non-breaking half space and replace with a plain space
- Remove the space between ± symbol and the numeral.

jwwojak committed:
Changes address review comments

- removing strange half-spaces
- use en dash to separate volume ranges
- remove space between ± and numeral

ecormany commented on docs/flex/docs/system-description/pipettes.md:
```diff
@@ -40,130 +53,181 @@ Because of their wide overall range, they can use multiple sizes of tips, which
-      <td>50 µL</td>
-      <td>5 µL</td>
-      <td>5.00%</td>
+      <td rowspan="4"><b>5-1000 µL</b></td>
```
last hyphen 🔬

jwwojak replied:
I'm going to start using `&ndash;` and then have Gemini change them to –  🙈  😆  

I found a missing ± as well. Commit incoming!

jwwojak committed:
One more missing ± in the data table

that should be it

jwwojak committed:
Once more unto the en dash dear friends!

Else we stop up this breech with our English [punctuation] dead.
